tdrs-backend/tdpservice/reports/models.py

```diff
@@ -54,63 +52,53 @@ class Meta:
             )
         ]
 
-    quarter = models.CharField(max_length=16,
-                               blank=False,
-                               null=False,
-                               choices=Quarter.choices)
+    quarter = models.CharField(
+        max_length=16, blank=False, null=False, choices=Quarter.choices
+    )
     year = models.CharField(max_length=16, blank=False, null=False)
-    section = models.CharField(max_length=32,
-                               blank=False,
-                               null=False,
-                               choices=Section.choices)
+    section = models.CharField(
+        max_length=32, blank=False, null=False, choices=Section.choices
+    )
 
     version = models.IntegerField()
 
-    user = models.ForeignKey(User,
-                             on_delete=models.CASCADE,
-                             related_name="user",
-                             blank=False,
-                             null=False)
-    stt = models.ForeignKey(STT,
-                            on_delete=models.CASCADE,
-                            related_name="sttRef",
-                            blank=False,
-                            null=False)
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="user", blank=False, null=False
+    )
+    stt = models.ForeignKey(
+        STT, on_delete=models.CASCADE, related_name="sttRef", blank=False, null=False
+    )
 
     @classmethod
     def create_new_version(self, data):
         """Create a new version of a report with an incremented version."""
         # EDGE CASE
         # We may need to try to get this all in one sql query
         # if we ever encounter race conditions.
-        version = (self.find_latest_version_number(year=data['year'],
-                                                   quarter=data['quarter'],
-                                                   section=data['section'],
-                                                   stt=data['stt']) or 0) + 1
+        version = (
```

Comment:
Ultra nit but this might be a little easier to read:
```
latest_version = self.find_latest_version_number(**data) or 0
version = latest_version + 1
```
That relies on the assumption that `data` doesn't include any keys other than `year`, `quarter`, `section` and `stt`. If that is not correct then the `find_latest_version_number` would need an extra argument, `**_` so it can safely ignore the extra keyword arguments.